### PR TITLE
[TRAFODION-2973] Add syntax and examples of *[LAST N] (SELECT Statement)* in *Trafodion SQL Reference Manual* 

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc [hunk 1]
```diff
@@ -6989,20 +6989,22 @@ By abstracting the complicated parts of the query into simpler, separate and log
...
+You must enclose `ANY _N_`, `FIRST _N_` or `LAST _N_` in square brackets ([]). The quotation marks ("") around each square bracket in
 the syntax diagram indicate that the bracket is a required character that you must type as shown (for example, [ANY 10]
 or [FIRST 5]). Do not include quotation marks in ANY or FIRST clauses.
 +
 [FIRST _N_] is different from [ANY _N_] only if you use ORDER BY on any of the columns in the select list to sort the
 result table of the SELECT statement. _N_ is an unsigned numeric literal with no scale. If _N_ is greater than the number
 of rows in the table, all rows are returned. [ANY _N_] and [FIRST _N_] are disallowed in nested SELECT statements and on
 either side of a UNION operation.
++
+`[LAST _N_]` performs a full table scan and calculates elapsed time. The _N_ must be 0 or 1. `[LAST _0_]` does not return any rows. `[LAST _1_]` returns only the last qualified row.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: "... performs a full table scan ..." This is not quite correct. Our query plan might have key predicates that limit us to less than a full table scan, for example. A more accurate description might be "... performs the entire query ...".

Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: Thanks for your explanations! Your comments have been incorporated. :)

**Comment by traflm** [comment]
Just for your information, below is the commands I used to merge the PR as a committer:

The pull request ID is 1458 for this one, and I assume your github account is liuyutw
-----------------------------------------------------------------------------
git clone https://github.com/apache/trafodion 
git remote add apache https://LiuYuTW@git-wip-us.apache.org/repos/asf/trafodion.git

git fetch apache
git checkout -b mrg_1458 origin/master

git fetch origin +refs/pull/1458/head

git merge --no-ff -m "[TRAFODION-2973] Add syntax and examples of *[LAST N] (SELECT Statement)* in *Trafodion SQL Reference Manual  " FETCH_HEAD

below two commands is just for checking the correctness, you can ignore them:
git log apache/master..HEAD
git diff apache/master HEAD

git push apache HEAD:default

**Comment by sandhyasun** [comment]
Yes the merge info is also documented here. For Ref Manual, there is one other step - i.e publishing. See this link : https://cwiki.apache.org/confluence/display/TRAFODION/Merge+Changes

**Comment by LiuYuTW** [comment]
Many thanks! :) @traflm @sandhyasun

**Pull request merged**